src/url-params.ts

```diff
@@ -98,6 +101,15 @@ export class AnalyticsUrlParams {
 		return Array.from(res);
 	}
 
+	/**
+	 * @return an array of passed Device Ids via the URL param if there was any
+	 */
+	getPassedDeviceIds() {
```

Comment:
I believe this is now `getPassedDeviceId` (singular)

Reply:
Yes you are right, let me fix it asap